src/TwigComponent/composer.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "symfony/ux-twig-component",
+    "type": "symfony-bundle",
+    "description": "Twig components for Symfony",
+    "keywords": [
+        "symfony-ux",
+        "twig",
+        "components"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\TwigComponent\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\TwigComponent\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=7.2.5",
```

Comment:
just a question, why the other is 7.4 and this ine 7.2?

Reply:
Excellent question! We originally targeted 7.2, but it became a burden on the other one. A final decision needs to be made - and these should probably be consistent.

Reply:
v7.4 would be top 👍 
v7.2 (and v7.3) are eol or soon to be :)